parachain-tracer/src/prometheus.rs

```diff
@@ -88,11 +90,19 @@ impl Metrics {
 	}
 
 	pub(crate) fn on_block(&self, time: f64, para_id: u32) {
+		const STANDARD_BLOCK_TIME: f64 = 6.0;
 		if let Some(metrics) = &self.0 {
 			metrics
 				.relay_block_times
 				.with_label_values(&[&para_id.to_string()[..]])
```

Comment:
Not really related to this PR, but we should only compute relay chain block time only once. Currently we do this for every para.